src/ucs/sys/sys.h

```diff
@@ -600,6 +600,27 @@ ucs_status_t ucs_sys_get_boot_id(uint64_t *high, uint64_t *low);
  */
 ucs_status_t ucs_sys_readdir(const char *path, ucs_sys_readdir_cb_t cb, void *ctx);
 
+/**
```

Comment:
What if there are multiple matches?
1. Is it possible in the specific `bond` case?
2. As this func is exposed in a header file, need to document its behavior in the general case.

Reply:
Thanks Leonid. Do you mean we search `lower_bond*` with high priority? If not found, then use the first one matched `lower_*`.